### DISCUSSION TITLE
Installation in Windows

### DISCUSSION THREAD

**Discussion by Rajatpareek** [InteractiveComputerGraphics/SPlisHSPlasH#183] [Q&A] [answered]
Hey there, I am facing an issue during installation. After configuration and generation in Cmake, when I am opening project its not creating SPHSimualator.exe anywhere also no bin folder created. Instead of this I am getting different file type simulators like vcproj etc. in build.
Can someone please guide me for installation.
Thank you in advance

**Comment by WeilerMarcel** [comment, marked as answer]
Hi @Rajatpareek,
it sounds like you only configured the cmake project and generated the Visual Studio project file. You will have to open that project in Visual Studio and build the executable afterwards.

![cmake](https://user-images.githubusercontent.com/18287644/171366435-38a1ed6b-ee1c-4b47-9e45-ae08e61ecf35.png)

Alternatively you can use CMake from the command line to drive the Visual Studio compiler. To do that, switch into your build directory after configuring/generating the project and call `cmake --build . --config Release`.

**Comment by Rajatpareek** [comment]
Hi @WeilerMarcel thank you for your detailed & kind suggestion. It worked, but I am getting SPHSimulator_d.exe. Is that similar to SPHSimulator.exe ?

**Comment by janbender** [comment]
This is the debug version. To get the optimized version you have to build the project in release mode.

**Comment by Rajatpareek** [comment]
Thank you for your help. Will try this :)

**Comment by Rajatpareek** [comment]
Is there any possible way to install SPlisHSPlasH in windows without visual studio?
Actually I am having some PC limitations so...
Thank you

**Comment by janbender** [comment]
A simple solution is to install the Python version using pip:

pip install pysplishsplash

You only need a current Python installed on your computer.
